tests/test_38_organizations.php

```diff
@@ -0,0 +1,92 @@
+
+       self::$api = new BigML(["project" => self::$project->resource]);
+    }
+
+    public static function tearDownAfterClass() {
+       self::$api_org->delete_all_project_by_name(self::$test_name);
+    }
+
+    /*  Scenario: Successfully creating a prediction: */
+
+    public function test_scenario1() {
+
+        $data = array(array("filename"=>  self::$data_localfile,
+                            "data_input" => array('petal width'=> 0.5),
+                            "objective" => "000004",
+                            "prediction" => "Iris-setosa")
+        );
+        
+        foreach($data as $item) {
+           print "\nSuccessfully creating a prediction:\n";
+           print "Given I create a data source uploading a ". $item["filename"]. " file\n";
+           $source = self::$api->create_source($item["filename"],
+                                               $options = array(
+                                                  'name'=>'local_test_source'));
+           $this->assertEquals(BigMLRequest::HTTP_CREATED, $source->code);
+           $this->assertEquals(1, $source->object->status->code);
+           
+           print "And I wait until the source is ready\n";
+           $resource = self::$api->_check_resource($source->resource, null, 3000, 30);
+           $this->assertEquals(BigMLRequest::FINISHED, $resource["status"]);
+           
+           print "And I create dataset with local source\n";
+           $dataset = self::$api->create_dataset($source->resource);
+           $this->assertEquals(BigMLRequest::HTTP_CREATED, $dataset->code);
+           $this->assertEquals(BigMLRequest::QUEUED, $dataset->object->status->code);
+           
+           print "And I wait until the dataset is ready\n";
+           $resource = self::$api->_check_resource($dataset->resource, null, 3000, 30);
+           $this->assertEquals(BigMLRequest::FINISHED, $resource["status"]);
+           
+           print "And I create model\n";
+           $model = self::$api->create_model($dataset->resource);
+           $this->assertEquals(BigMLRequest::HTTP_CREATED, $model->code);
+           
+           print "And I wait until the model is ready\n";
+           $resource = self::$api->_check_resource($model->resource, null, 3000, 30);
+           $this->assertEquals(BigMLRequest::FINISHED, $resource["status"]);
+           
+           print "When I create a prediction for " . json_encode($item["data_input"]) . "\n";
+           $prediction = self::$api->create_prediction($model, $item["data_input"]);
+           $this->assertEquals(BigMLRequest::HTTP_CREATED, $prediction->code);
+           
+           print "Then the prediction for " . $item["objective"] . " is " . $item["prediction"];
+           
+           $this->assertEquals($item["prediction"],
+                               $prediction->object->prediction->{$item["objective"]});
+           
```

Comment:
you should check that these objects are created in the organization's project and also, previously, that the project is created inside of the organization.